test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/TagHelperOutputTest.cs

```diff
@@ -154,7 +115,7 @@ public void GenerateStartTag_ReturnsNothingIfWhitespaceTagName()
             var output = tagHelperOutput.GenerateStartTag();
 
             // Assert
-            Assert.Empty(output);
+            Assert.Null(output);
```

Comment:
@NTaylorMullen I agree w/ not creating new `TagHelperContent` instances.  but this method returns a `string` right now.  that's weird: it and `GenerateEndTag()` should return the same type as the `Generate*Content()` methods.